tests/constants/dockercli/cmd.go

```diff
@@ -35,9 +35,27 @@ const (
 	// KillDocker kill docker
 	KillDocker = "pkill -9 dockerd "
 
+	// GetVDVSPlugin gets vDVS plugin info
+	GetVDVSPlugin = docker + "plugin list --no-trunc | grep vsphere"
+
+	// KillVDVSPlugin kills vDVS plugin
+	KillVDVSPlugin = "docker-runc kill "
+
+	// StartVDVSPlugin starts the vDVS plugin
+	StartVDVSPlugin = docker + " plugin enable vsphere"
```

Comment:
minor: better to create a separate constant for `vsphere` (plugin name) .. any plugin installation with an alias other than `vsphere` fails our code. It is better to have a hook to modify plugin name easily.